add some new pandunia words

jkunimune opened the pull request:
New words: gorizmi, also, hobi, lover, al, lakin, konhave, konstru bash, kopihak, binaya, binatur, kurve, detal, disjiu, edit, editer, and koretifa.

I also added a new meaning for fikre and added dictionary entries for the words monta, plis, and kom, which were already on the wobsite.  Also deleted the duplicate PWN:ask.v.02 definition.

I added a couple new temporary IDs; I don't see anything in Github actions about finding and renaming temporary IDs, and I can't easily run the shell script because I'm in Windows.  Is it easy for you to periodically run the shell scripts on your end?  If it's a pain, I can install WSL and run them myself.

In general, would you rather I submit new words directly as pull requests like this, or post them as proposals to the Discord or subreddit?

jkunimune committed:
add some new concept definitions

jkunimune committed:
add some new concept definitions

jkunimune committed:
add English translations for new concepts

jkunimune committed:
add English translations for new concepts

jkunimune committed:
Merge branch 'master' of https://github.com/jkunimune/panlexia

barumau commented:
I can run the scripts myself periodically. It's no problem for me because there are only a 2-3 of us. But of course I would like spend more time on contents and less on editorial work.

Now in now in this pull request you can see the results of GitHub actions. Just click `All checks have passed` and then `Checks for changeset / check-changes (pull_request)`. Then there is a section called `Find and rename temporary concept ids` where the temporary ids are listed.

You should be able to see the same things in your forked repository. First you should create a branch by `git checkout -b my_new_branch`. Make your changes, push the branch by `git push origin my_new_branch`, and create a pull request in GitHub website.

In any case it would be good to work with branches, because then you/we could work with several independent changes at the same time. See especially step 5 in https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork

barumau commented on dict/P/pandunia.tsv:
```diff
@@ -1552,3 +1552,30 @@ PWN:zero.s.01		siro	siro	ara: صِفْر‎ (ṣifr), swa:sifuri, hau:sifili, hi
 PWN:zinc.n.01		zinke	zink'e	eng:zinc, fra:zinc, spa:cinc, por:zinco, rus:цинк, zho:锌 (xīn), ben:জিঙ্ক, may:seng, swa:zinki
 PWN:zirconium.n.01		zirkonium	zirkon·ium	eng:zirconium, fra:zirconium, spa:circonio, por:zircónio, rus:цирконий, jpn:ジルコニウム, kor:지르코늄, vie:ziriconi, hin:जर्कोनियम, ben:জিরকোনিয়াম, may:zirkonium, swa:zirikoni, ara: زيركونيوم
 PWN:zygote.n.01		zugot	zug·ot	
+PLX:algorithm.n.01		gorizmi	gorizmi	ara:خَوَارِزْمِيَّة, eng: algorithm, spa:algoritmo, rus:алгори́тм, fas:الگوریتم, hin:अल्गोरिद्म, may:algoritma, jpn:アルゴリズム
```
I prefer the form *gorizme*, because then we could derive *gorizmik* ('algorithmic').

barumau commented on dict/P/pandunia.tsv:
```diff
@@ -1552,3 +1552,30 @@ PWN:zero.s.01		siro	siro	ara: صِفْر‎ (ṣifr), swa:sifuri, hau:sifili, hi
 PWN:zirconium.n.01		zirkonium	zirkon·ium	eng:zirconium, fra:zirconium, spa:circonio, por:zircónio, rus:цирконий, jpn:ジルコニウム, kor:지르코늄, vie:ziriconi, hin:जर्कोनियम, ben:জিরকোনিয়াম, may:zirkonium, swa:zirikoni, ara: زيركونيوم
 PWN:zygote.n.01		zugot	zug·ot	
+PLX:algorithm.n.01		gorizmi	gorizmi	ara:خَوَارِزْمِيَّة, eng: algorithm, spa:algoritmo, rus:алгори́тм, fas:الگوریتم, hin:अल्गोरिद्म, may:algoritma, jpn:アルゴリズム
+PWN:besides.r.02		also	also	eng:also
```
Let's use the word *plus* as in old Pandunia because it's more international than *also*.

barumau commented on dict/P/pandunia.tsv:
```diff
@@ -1552,3 +1552,30 @@ PWN:zero.s.01		siro	siro	ara: صِفْر‎ (ṣifr), swa:sifuri, hau:sifili, hi
+PLX:algorithm.n.01		gorizmi	gorizmi	ara:خَوَارِزْمِيَّة, eng: algorithm, spa:algoritmo, rus:алгори́тм, fas:الگوریتم, hin:अल्गोरिद्म, may:algoritma, jpn:アルゴリズム
+PWN:besides.r.02		also	also	eng:also
+TMP:00432689.n		hobi	hobi	eng:hobby, deu:Hobby, spa:hobby, rus:хобби, tur:hobi, jpn:ホビー, may:hobi
+PWN:amateur.n.01		hobitiste	hobi·tist'e	
```
This is correct by the current rules but I think that *hobi·ist'e* would be better. After all, there are international words like *altruism*, *lamaism*, *egoism* and *Maoism* where there isn't any connecting consonant before the suffix.

barumau commented on dict/P/pandunia.tsv:
```diff
@@ -1552,3 +1552,30 @@ PWN:zero.s.01		siro	siro	ara: صِفْر‎ (ṣifr), swa:sifuri, hau:sifili, hi
+PWN:amateur.n.01		lover	lov·er	
+TMP:09622302.n		lover	lov·er	
+PWN:amount.n.02		monta	monta	eng:amount, spa:monto
+PWN:ask.v.02		plis	plis	eng:please
```
There is already *ples* at https://github.com/barumau/panlexia/blob/b3d79a26b0d1a31e7e6181fc7eae26fad4377d48/dict/P/pandunia.tsv#L1525. I decided to use *ples* in the sense 'to desire, to wish' and keep *ching* from version 2 in the sense 'to request'.

Why *ples* and not *plis*, you might ask. It is for similarity to French.

- Pandunia: *if tu/yu ples*
- English: *if you please*
- French: *s'il te/vous plaît*

barumau commented on dict/P/pandunia.tsv:
```diff
@@ -1552,3 +1552,30 @@ PWN:zero.s.01		siro	siro	ara: صِفْر‎ (ṣifr), swa:sifuri, hau:sifili, hi
+TMP:09622302.n		lover	lov·er	
+PWN:amount.n.02		monta	monta	eng:amount, spa:monto
+PWN:ask.v.02		plis	plis	eng:please
+TMP:02269286.a		al	al	eng:all, deu:all
```
There is already the word for this, *hol*. It combines Greek words (*holo-*, *holism*, etc.) and English pronunciation /ɔ:l/.

barumau commented on dict/P/pandunia.tsv:
```diff
@@ -1552,3 +1552,30 @@ PWN:zero.s.01		siro	siro	ara: صِفْر‎ (ṣifr), swa:sifuri, hau:sifili, hi
+PWN:ask.v.02		plis	plis	eng:please
+TMP:02269286.a		al	al	eng:all, deu:all
+PWN:both.s.01		al du	al du	
+PWN:however.r.01		lakin	lakin	ara:لٰكِن, swa:lakini, hin:लेकिन, tur:lakin
```
*lakin* is a good word! Basically it's just a longer way to say *but*.

barumau commented on dict/P/pandunia.tsv:
```diff
@@ -1552,3 +1552,30 @@ PWN:zero.s.01		siro	siro	ara: صِفْر‎ (ṣifr), swa:sifuri, hau:sifili, hi
+PWN:consider.v.04	fikre	fikr'e
+PLX:conlang.n.01	konstru bash	konstru bash	
+PWN:copyright.n.01	kopihak	kopi·hak	
+PWN:creation.n.01	binaya	bina·ya	
```
I would use the unlisted root *krea* for this sense as a higher form of making.

Then the suffix is *-(t)ion* when we talk about the act. *-ia/-ya* means collectives, so *kreaya* would mean everything that has been created.

barumau commented on dict/P/pandunia.tsv:
```diff
@@ -1552,3 +1552,30 @@ PWN:zero.s.01		siro	siro	ara: صِفْر‎ (ṣifr), swa:sifuri, hau:sifili, hi
+PWN:curve.n.01	kurve	kurv'e	eng: curve, spa:curva, por:curva, fra:courbe, deu:Kurve, rus:крива́я
+PWN:curved.a.01	kurve	kurve	
+PWN:detail.n.02	detal	detal	eng:detail, spa:detalle, por:detalhe, fra:détail, deu:Detail, rus:дета́ль, tur:detay
+PWN:distribute.v.04	disjiu	dis·jiu	
```
The meaning of *jiu* is `PWN:give.v.33   be flexible under stress of physical force`.
You must mean *don* ̀`PWN:give.v.01   cause to have, in the abstract sense or physical sense`.

All the rest are good. :)

barumau commented on concepts/E/eng-definition.tsv:
```diff
@@ -7505,3 +7504,9 @@ PWN:zirconium.n.01	Zirconium, the chemical element with atomic number 40 and sym
+TMP:09622302.n	a person who loves someone or is loved by someone
+TMP:00432689.n	an auxiliary activity
+TMP:02269286.a	quantifier, used to refer to the entire quantity or extent of something; entity, universal pronoun; quantifier, universal
+PWN:both.s.01	two considered together; the two
```
I already extracted the definition for this from Wordnet. It's at https://github.com/barumau/panlexia/blob/master/concepts/E/eng-definition.tsv#L1026

I think you should rebase your branch.

barumau reviewed (changes requested):
It's great that you are working for Pandunia again! I really appreciate it.

jkunimune committed:
add some new pandunia words

gorizmi, also, hobi, lover, al, lakin, konhave, konstru bash, kopihak, binaya, binatur, kurve, detal, disjiu, edit, editer, and koretifa.

I also added a new meaning for fikre and added dictionary entries for the words monta, plis, and kom, which were already on the wobsite.  also deleted the duplicate PWN:ask.v.02 definition.

jkunimune committed:
Merge branch 'master' of https://github.com/jkunimune/panlexia

jkunimune committed:
Merge remote-tracking branch 'upstream/master'

jkunimune committed:
update words based on feedback

use existing roots ching, hol, and krea, and use different suffixes on gorizme, hobiiste, and kreation.

jkunimune commented:
Okay, I got a little confused with the rebase (I thought Iwas up to date when I started this branch but I guess not) but it seems to be good and conflict-free now.  I updated those words based on your comments.

barumau reviewed (approved):
I approve these changes, and I will handle the TMP ids myself.
